nuxeo-document.html

```diff
@@ -138,8 +138,8 @@
 
             /** The `content enricher` of the resource. */
             enrichers: {
-              type: String,
-              value: '',
+              type: Object,
+              value: {},
```

Comment:
Shouldn't these changes be applied to `nuxeo-audit-page-provider.html` and `nuxeo-page-provider.html` as well?